db/view/view_building_worker.cc

```diff
@@ -374,7 +374,7 @@ future<> view_building_worker::run_view_building_state_observer() {
             // A batch could finished its work while the worker was
             // updating the state. In that case we should do another iteration.
             if (!_state.some_batch_finished) {
-                co_await _vb_state_machine.event.wait();
+                co_await _cv.wait();
```

Comment:
I would say that having this patch separately makes little sense. Please, merge it to previous one and udpate the patch comment with more detailed explanation why it's better to use dedicated c.v. rather than state machine's one

Reply:
no more relevant